pkg/kiln/io.go

```diff
@@ -295,3 +300,22 @@ func GetExampleDockerfile(dockerInfo *DockerInfo) (buf *bytes.Buffer, err error)
 
 	return buf, nil
 }
+
+// DetermineBaseImage consumes a given runtime selection and determines the base image for the Dockerfile
+func DetermineBaseImage(runtimeSelection string) (baseImage string, err error) {
+	runtimeSplit := strings.Split(runtimeSelection, ":")
+
+	switch runtimeSplit[0] {
```

Comment:
I had the same idea, to use a map to store the supported runtimes, but I wasn't sure how much more effective that would be than this given we would still need to manually update the map, and also possibly concat the base image with the desired version (i.e. `node:4`, `node:5.1.2`, `java:7`) as is done here. It is something to explore, and if my mental model is off-base or you had another perspective, let me know.